### PR TITLE
feat(playbooks): wkthmltox for Ubuntu 20

### PR DESCRIPTION
**Pull request by nikunj1222**
https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.focal_amd64.deb

Updating file to support Ubuntu20.04 LTS

### CODE REVIEW

File: bench/playbooks/roles/wkhtmltopdf/tasks/main.yml [hunk 1]
```diff
@@ -19,7 +19,15 @@
     state: present
     force: yes
   when: ansible_os_family == 'Debian'
+
```
Comment (gavindsouza) [inline bench/playbooks/roles/wkhtmltopdf/tasks/main.yml:22]:
```suggestion
```

File: bench/playbooks/roles/wkhtmltopdf/tasks/main.yml [hunk 2]
```diff
@@ -20,6 +20,13 @@
     force: yes
   when: ansible_os_family == 'Debian'
 
+- name: download wkthmltox Ubuntu 20
+  get_url:
+    url: https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.focal_amd64.deb
+    dest: /tmp/wkhtmltox.deb
+    checksum: "sha256:{{ 'db48fa1a043309c4bfe8c8e0e38dc06c183f821599dd88d4e3cea47c5a5d4cd3' if ansible_architecture == 'x86_64' else '1f5ac84c1cb25e385b49b94a04807d60bf73da217bc6c9fe2cbd1f0a61d33f63' }}"
```
Comment (Alchez) [inline bench/playbooks/roles/wkhtmltopdf/tasks/main.yml:27]: Will this checksum work? Seems to be the same as the one for 18?

Comment (gavindsouza) [reply bench/playbooks/roles/wkhtmltopdf/tasks/main.yml:27]: Contents of **SHA256SUMS** file from https://wkhtmltopdf.org/downloads.html
```
db48fa1a043309c4bfe8c8e0e38dc06c183f821599dd88d4e3cea47c5a5d4cd3 *wkhtmltox_0.12.5-1.bionic_amd64.deb
1f5ac84c1cb25e385b49b94a04807d60bf73da217bc6c9fe2cbd1f0a61d33f63 *wkhtmltox_0.12.5-1.bionic_i386.deb
c60e75fef5bfa1e79983919ffb47b40dcfbb49d121a510f11ca4b2a2603c00f1 *wkhtmltox-0.12.5-1.centos6.i686.rpm
17bff4966143d240a126b6cc414c6f79aa2106c0c97c772228e84d685221c25f *wkhtmltox-0.12.5-1.centos6.x86_64.rpm
1030279ac4b5b15dda04de2587b2a1942bde1c78aa1837dfec4ddcbea426721f *wkhtmltox-0.12.5-1.centos7.i686.rpm
ac4f909b836fa1fc0188d19a1ab844910f91612e9ccefcb5298aa955a058ffe4 *wkhtmltox-0.12.5-1.centos7.x86_64.rpm
2583399a865d7604726da166ee7cec656b87ae0a6016e6bce7571dcd3045f98b *wkhtmltox_0.12.5-1.jessie_amd64.deb
3a6969f3ed207a805092e05794644eb9e152aaa6518e9204c819fa318947a8a8 *wkhtmltox_0.12.5-1.jessie_i386.deb
dadab659ff2ffb3316063800ff1885f989a53e85535049822ba33cfbf37975e9 *wkhtmltox-0.12.5-1.macos-carbon.pkg
2718c057249a133fe413b3c8cfb33b755a2e18a8e233329168f1af462cd6de5f *wkhtmltox-0.12.5-1.macos-cocoa.pkg
894644b32c767dc267513f381d06ec5783c2cbaa3e9e139adc1e04d3facf4561 *wkhtmltox-0.12.5-1.msvc2015-win32.exe
b5a60b8801a599b9097c5cc8b0d2b18d1709b119b9d88b960f1f38fad498dcd8 *wkhtmltox-0.12.5-1.msvc2015-win64.exe
eb00f88ceed2941430cb05bc91da4e753e2319188b8e891735ad141aad668ae8 *wkhtmltox-0.12.5-1.mxe-cross-win32.7z
d9c4717318439f7576ce201de145f0b71ffdd209faadec8dcfb1be088eae6e71 *wkhtmltox-0.12.5-1.mxe-cross-win64.7z
a849511325c8eb72fe5116ec236a0a6e881e6a539d4e32ebef41ac1c385859c9 *wkhtmltox_0.12.5-1.raspbian.stretch_armhf.deb
1140b0ab02aa6e17346af2f14ed0de807376de475ba90e1db3975f112fbd20bb *wkhtmltox_0.12.5-1.stretch_amd64.deb
5b2d15e738ac479e7a8ca6fd765f406c3684a48091813520f87878278d6dd22a *wkhtmltox_0.12.5-1.stretch_i386.deb
2a3d1fe80da0dbc69da56cf90a3d0ec2786d1b919be29527630d609fea4a6b7c *wkhtmltox_0.12.5-1.trusty_amd64.deb
582e02881e4bc6be9aaa634da1fe8c02d3233fb57f6daab9efa137edb812dd3b *wkhtmltox_0.12.5-1.trusty_i386.deb
df203cee4dc9b3efb8d0cd6fc25fa819883224f50c75b76bd9c856903711dc14 *wkhtmltox_0.12.5-1.xenial_amd64.deb
27b6edafee099b87b2911cc68b780e79cffed3948bb5a074e8ea1cf8820da156 *wkhtmltox_0.12.5-1.xenial_i386.deb
```

You're right! focal's checksums aren't posted yet? I only checked the architecture and version. My bad. This shouldn't work.

Comment (nikunj1222) [reply bench/playbooks/roles/wkhtmltopdf/tasks/main.yml:27]: can we ignore them for a while...till we receive the new checksum..

Comment (gavindsouza) [reply bench/playbooks/roles/wkhtmltopdf/tasks/main.yml:27]: @nikunj1222 remove the checksum key for now. We have a couple other actions in the file that install `wkhtmltopdf` without the checksum too.

Comment (nikunj1222) [reply bench/playbooks/roles/wkhtmltopdf/tasks/main.yml]: Removed checksum from ubuntu20.

**Pull request merged**